### PR TITLE
Fix vat-subui-fail spec

### PR DESCRIPTION
**Pull request by gtrepta**

### CODE REVIEW

**Comment by ehildenb** [comment]
@gtrepta you likely need to increases the amount of time that we allocate for testing on this branch.

File: tests/failing-symbolic.haskell [hunk 1]
```diff
@@ -18,9 +18,7 @@ tests/specs/mcd/pot-join-pass-rough-spec.k
 tests/specs/mcd/vat-fold-pass-rough-spec.k
 tests/specs/mcd/vat-fork-diff-pass-rough-spec.k
 tests/specs/mcd/vat-frob-diff-zero-dart-pass-rough-spec.k
-tests/specs/mcd/vat-move-diff-rough-spec.k
 tests/specs/mcd/vat-slip-pass-rough-spec.k
```
Comment (ehildenb) [inline tests/failing-symbolic.haskell:21]:
```suggestion
tests/specs/mcd/vat-move-diff-rough-spec.k
tests/specs/mcd/vat-slip-pass-rough-spec.k
```

File: tests/specs/mcd/verification.k [hunk 2]
```diff
@@ -93,6 +93,43 @@ module LEMMAS-MCD-HASKELL [symbolic, kore]
...
+    // (declare-const x Int)
+    // (declare-const y Int)
+    //
+    // (assert (uint256 x))
+    // (assert (sint256 y))
+    // (assert (<= y 0))
+    // (assert (not (uint256 (- x y))))
+    //
+    // (assert (not (= (chop (- x y)) (- x (+ y pow256)))))
+    // (check-sat)
```
Comment (ehildenb) [inline tests/specs/mcd/verification.k:126]: What was the output this supplied? It should say `unsat`, then we know that the lemma is true.

**Review by ehildenb — APPROVED** [review]

File: tests/specs/mcd/verification.k (hunk 2 above)
Comment (ehildenb) [reply tests/specs/mcd/verification.k:126]: I double-checked that it gave unsat on my machine.

Comment (gtrepta) [reply tests/specs/mcd/verification.k:126]: Yep, it was unsat.

**Pull request merged**